integrations/amazonKeys/cli/init.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash -e
+readonly IFS=$'\n\t'
+readonly ROOT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+readonly SCRIPT_NAME="$( basename "$0" )"
+
+export RESOURCE_NAME=""
+export SCOPE=""
+
+print_help() {
+  echo "
+  Usage:
+    $ROOT_DIR/$SCRIPT_NAME <resource_name> [scope]
+  "
+}
+
+parse_args() {
+  if [ "$#" -gt 0 ]; then
+    RESOURCE_NAME=$1
+    SCOPE=$2
+    if [ "$SCOPE" == "" ]; then
+      SCOPE="configure"
+    fi
+  else
+    print_help
+    exit 1
+  fi
+}
+
+setup() {
+  echo "Setting up $SCOPE for resource $RESOURCE_NAME."
+}
+
+main() {
+  parse_args "$@"
+  setup
+}
+
+main "$@"
```

Comment:
actually, we can just store the args in a global variable at the top like `readonly ARGS=$@` and then any function can process. we wont have to pass them around in arguments that way

Reply:
Okay, I'll change this.